core.js

```diff
@@ -44,7 +44,7 @@ exports.init = function (dir, config) {
     })
 
     // helper for rooms to allow connecting to friends directly
-    SSB.net.friends = {
+    /*SSB.net.friends = {
```

Comment:
This should stay in

Reply:
It crashed when it was in, which is why I commented it out.  I don't remember the details now, but I think it was ssb-suggest was looking for the presence of ssb-friends and trying to use methods this didn't define, which led to a crash.  This was the only thing defining an ssb-friends-alike, so by taking it out, ssb-suggest didn't try to use the nonexistent methods.  Again, don't remember the details, but it didn't work without this being gone.

Reply:
Sure enough, it was because of ssb-suggest.  I added it back in but also stuck on a stub function for what ssb-suggest looks for so that it at least doesn't crash.